internal/driver/registry_default.go

```diff
@@ -114,6 +114,10 @@ func (r *RegistryDefault) Tracer(ctx context.Context) *otelx.Tracer {
 		r.tracer = t
 	}
 
+	if r.tracer.Tracer() == nil {
```

Comment:
How can we run into this case? :thinking: 
https://github.com/ory/x/blob/32ada8266273e76e35cc692a3999eb966cbeab8b/otelx/otel.go#L64-L65

Reply:
Ah I forgot I'd added a check for it already. I'll remove it here.